ProjectEureka/project-eureka-flutter
# PE-78  | Refactor New Question Form for Reuse as a New Answers Form

**What I Did:**
- Refactored the New Question Form so we can reuse it with new answers.
- Renamed some corresponding files so its not restricted to just questions.

**How to Test:**
1. Clone branch and `flutter pub get`
2. Click the More Details Button on the Home Screen and then Answer on the bottom, you should be able to see a form that will let you add an answer.
3. Test it by filling out the form, include a picture. Note this is not connected to the backend.
4. Verify that when submitted, you can get sent to a confirmation screen with your answer and the image you took is there. 
5. Now verify that Create New Question still works as intended.
